english/memories/ground-hog/kitchen/garden/beans.md

```diff
@@ -0,0 +1,5 @@
+Ah! Good choice. Beans are high in protein and have virtually no fat. They also have more fiber than most whole grain foods.
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.